addons/mail/static/src/discuss/core/common/thread_actions.js

```diff
@@ -189,7 +189,7 @@ threadActionsRegistry
     .add("mark-read", {
         condition: (component) =>
             component.thread?.selfMember?.message_unread_counter > 0 &&
-            !component.thread?.mute_until_dt,
+            !component.thread?.selfMember?.mute_until_dt,
```

Comment:
Same here: this is true if `thread.selfMember` is `undefined`. We might want to check for `thread.selfMember` first.